connection.js

```diff
@@ -1255,8 +1255,13 @@ class Connection {
             return this.respond(501, "No command given");
         }
 
-        require('./server').sendToMaster(command, results);
-        return this.respond(250, "Command sent for execution. Check Haraka logs for results.");
+        const obj = require('./server').sendToMaster(command, results);
+        if (obj && obj.error)
```

Comment:
Sorry to nitpick, but a long time agree we settled on a single line or braces. Your choice.